runtime/nls/shrc/j9shr.nls

```diff
@@ -6869,3 +6869,11 @@ J9NLS_SHRC_SHRINIT_HELPTEXT_PRINTSTATS_HELP_V1.explanation=NOTAG
 J9NLS_SHRC_SHRINIT_HELPTEXT_PRINTSTATS_HELP_V1.system_action=
 J9NLS_SHRC_SHRINIT_HELPTEXT_PRINTSTATS_HELP_V1.user_response=
 # END NON-TRANSLATABLE
+
+J9NLS_SHRC_SHRINIT_HELPTEXT_NO_TIMESTAMP_CHECKS_V1=Completely turn off timestamp checking when finding classes from the shared cache.
```

Comment:
I'm thinking this should be expanded to include "Obsolete classes may be loaded from the cache."

Reply:
Added 
"Use this option only when you know there are no updates to the classes from the class paths or module paths in your application. Otherwise, obsolete classes may be loaded from the shared cache. If this scenario occurs, remove the this option."
to` J9NLS_SHRC_SHRINIT_HELPTEXT_NO_TIMESTAMP_CHECKS_V1.explanation`

Reply:
That doesn't work, the help text needs to be updated. It should be marked NOTAG, because the tag isn't printed. Without the tag, nobody will find the explanation. If you're concerned about length, the word "Completely" can be removed, also change from -> in.

Reply:
Fixed